quarry/web/static/templates/compiled.js

```diff
@@ -153,13 +150,16 @@ output += "\n";
 }
 ;
 }
+output += "\n";
+if(runtime.memberLookup((runtime.contextOrFrameLookup(context, frame, "extra")),"connection_id")) {
+output += "\n<button id=\"show-explain\" type=\"button\" class=\"btn btn-default btn-xs\">Explain</button>\n";
 ;
 }
 output += "\n";
 if(runtime.memberLookup((runtime.contextOrFrameLookup(context, frame, "extra")),"replag")) {
-output += "\n<div class=\"alert alert-warning\">\n  <h4 class=\"alert-heading\">Replication lag</h4>\n  <p class=\"mb-0\">The database on which this query was executed has a <a href=\"https://replag.toolforge.org/\">synchronization delay</a> with the wiki.\n      This can be caused by maintenance or incident on database, and should be resolved soon.<br>\n      The modifications that was made in last <b>";
```

Comment:
These are unrelated changes that were not made in this PR. Looks like the last time someone changed query-status.html, they missed running nunjucks precompile (understandable as there's no documentation on how to update it!). Perhaps we should add a CI step to ensure compiled.js matches the original files.